src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -910,6 +930,9 @@ private void InitEditor()
 
     private void InitEditorFresh()
     {
+        if (!transitionFinished)
+            return;
+
```

Comment:
Instead of this check, extend the auto-evo check like this:
```
        if (!CurrentGame.GameWorld.IsAutoEvoFinished())
        {
            ready = false;
            LoadingScreen.Instance.Show("Loading Microbe Editor", CurrentGame.GameWorld.GetAutoEvoRun().Status);
        }
        else if (!transitionFinished)
        {
             ready = false;
        }
        else
        {
            OnEditorReady();
        }
```